### PR TITLE
Update Crossplane to v1.9.1

### PR DESCRIPTION
**Pull request by uvegla**
Towards: https://github.com/giantswarm/giantswarm/issues/23933

### CODE REVIEW

**Comment by uvegla** [comment]
> Is it ready for review?

@kubasobon Yes, please check it. Even if we want to do more at this point, I would rather do it in another PR. Also it would be nice feedback if the direction is good because this is open for a while now.

File: helm/crossplane/values.yaml [hunk 1]
```diff
@@ -1,10 +1,108 @@
-image:
-  repository: crossplane/crossplane
-  tag: v1.7.0
-  pullPolicy: IfNotPresent
+giantswarm:
+  images:
+    registry: quay.io
```
Comment (kubasobon) [inline helm/crossplane/values.yaml:3]: Can we set the default repo to `docker.io`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,61 +1,22 @@
-# How to contribute
+## Update from upstream
```
Comment (kubasobon) [inline CONTRIBUTING.md:1]: Thank you for making this file usable. I'm always looking for instructions like these in `CONTRIBUTING.md`.

File: helm/crossplane/values.yaml [hunk 3]
```diff
@@ -1,10 +1,108 @@
...
+args: {}
 
 provider:
   packages: []
 
 configuration:
   packages: []
+
+imagePullSecrets:
+- dockerhub
```
Comment (kubasobon) [inline helm/crossplane/values.yaml:46]: Where can I find this secret?

**Review by kubasobon — COMMENTED** [review]
Left some comments, otherwise LGTM

File: helm/crossplane/values.yaml (hunk 3 above)
Comment (uvegla) [reply helm/crossplane/values.yaml:46]: It came from upstream. By chance, the very latest commit at the time in upstream removes it: https://github.com/crossplane/crossplane/commit/43c577253f6babcf5002944fa7e1d5e55cc74f45

Comment (uvegla) [reply helm/crossplane/values.yaml:46]: But latest release it is like this: https://github.com/crossplane/crossplane/blob/6b674f868403e92fa40d49a8a0385352d8d32e39/cluster/charts/crossplane/values.yaml.tmpl#L33-L34

**Pull request merged**